eZ/Bundle/PlatformBehatBundle/Context/RepositoryContext.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * File containing the Repository Context class for EzBehatBundle.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+namespace EzSystems\PlatformBehatBundle\Context;
+
+use eZ\Publish\API\Repository\Repository;
+use eZ\Publish\Core\Repository\Values\User\UserReference;
+
+/**
+ * Repository Context Trait.
```

Comment:
Please elaborate a bit about how it is used. I'd like to see at least one reference to a place where it's used, and a simplified example.